code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -79,6 +79,13 @@
 	end_sound = 'sound/machines/microwave/microwave-end.ogg'
 	volume = 90
 
+/datum/looping_sound/lathe_print
+	start_sound = 'sound/machines/lathe/protolathe_print.ogg'
+	start_length = 15
+	mid_sounds = list('sound/machines/lathe/protolathe_print2.ogg' = 1)
```

Comment:
These need better names. Call them what they are rather then nothing and 2. startup and processing or something.

Reply:
Also it's very clearly not just protolathes